### DISCUSSION TITLE
normal output of httpx is malformed

### DISCUSSION THREAD

**Discussion by 0xAwali** [projectdiscovery/httpx#558] [Q&A] [answered]
while adding options to httpx e.g. -title , -location etc with output file.txt then I tried to read file.txt I noticed it was malformed e.g. 

![1](https://user-images.githubusercontent.com/54332136/159578758-50790ec3-9a85-40a4-8d62-efe871545e7a.jpg)

the output should be like this

![2](https://user-images.githubusercontent.com/54332136/159578813-bf6c1757-db74-427f-9f18-bc41b6e19b2a.jpg)

so httpx -o malform the output

### httpx version:

v1.2.0

**Comment by wdahlenburg** [comment, marked as answer]
Try out the `-nc` flag for no color. It should remove the extra characters in the output.
